### PR TITLE
FEATURE: Add Eel String helper pregMatchAll

### PR DESCRIPTION
**Pull request by jonnitto**
This pull request add the Eel String helper `String.pregMatchAll`

### CODE REVIEW

File: Neos.Eel/Classes/Helper/StringHelper.php [hunk 1]
```diff
@@ -198,6 +198,31 @@ public function pregMatch($string, $pattern)
         return $matches;
     }
 
+      /**
+     * Perform a global regular expression match (PREG style)
```
Comment (gerhard-boden) [inline Neos.Eel/Classes/Helper/StringHelper.php]: Fix docblock indentation

**Comment by gerhard-boden** [comment]
Looks good except for one small indentation error

**Comment by kitsunet** [comment]
This might be just me, but I feel if you need to use pregMatchAll you might want to have a specialised helper for the use case or do some more in PHP. Just a hunch though, maybe I need to see some actual fusion code with this to see how that would look like.

**Comment by jonnitto** [comment]
In this case, I iterate over an SVG file string with symbols, and the goal was to get all the id in an array.

**Pull request merged**